src/Sentry.Serilog/SentrySink.cs

```diff
@@ -86,6 +96,7 @@ public void Emit(LogEvent logEvent)
                         Name = Constants.SdkName,
                         Version = NameAndVersion.Version
                     },
+                    Logger = logger,
```

Comment:
Would this work instead of having to change the code above to assing `logger = context`?
```suggestion
                    Logger = context,
```